warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__data_license.sql

```diff
@@ -0,0 +1,27 @@
+WITH
+
+idx AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__gtfs_dataset_schedule_guideline_index') }}
```

Comment:
Ok sorry for not catching this before -- I just double checked the guidelines directly and I think there are both RT & schedule versions of this -- so maybe it should be in the "all 4" category